sdk/identity/azure-identity/src/main/java/com/azure/identity/EnvironmentCredential.java

```diff
@@ -92,4 +105,28 @@ private boolean verifyNotNull(String... configs) {
         }
         return true;
     }
+
+    private void logAvailableEnvironmentVariables(String clientId, String tenantId, String clientSecret,
```

Comment:
Just an FYI about this, I logged this one for every credential, not just the environment credential, because environment variables can affect other credentials as well.

Reply:
I've added this to all the credentials that currently take advantage of environment variables, which are `EnvironmentCredential`, `ManagedIdentityCredential`, and `SharedTokenCredential`. It might be confusing for other credential types that in case of error, users might be misled to believe there's something wrong with their environment variables.